apps/masters/utils/table_defination.py

```diff
@@ -0,0 +1,155 @@
+from reportlab.lib.pagesizes import inch
+from reportlab.lib import colors
+from reportlab.platypus import Table, TableStyle, Paragraph
+from reportlab.lib.styles import getSampleStyleSheet
+from reportlab.platypus.flowables import Flowable
+
+
+
+
+
+def create_table_1(sno, sdate):
```

Comment:
Can we have a meaningful name here instead of create_table_1, create_table_2, create_table_3 etc....
ex: create_table_header, create_table_body etc.... like this

Please change everywhere

Reply:
Done